components/ps4_controller_pairing.mdx

```diff
@@ -1,21 +1,44 @@
-If your PS4 controller runs out of batteries, or you purchase a new one, you might want to re-pair your platform
-and controller. To do this, put the controller into pairing mode by pressing and holding the Share and PS buttons
-until the controller's LED flashes rapidly in white. Then SSH into the robot and run:
+If your PS4 controller runs out of batteries, or you purchase a new one, you might want to re-pair your platform and controller. 
+
+To do this:
+
+1. Make sure the `ds4drv` driver is installed, and the `ds4drv` daemon service is active and running:
+
+```
+sudo systemctl status ds4drv
+```
+
+2. If `ds4drv` is not installed, install it by running:
+
+```
+sudo apt-get install python-ds4drv
+```
+
+3. Put the controller into pairing mode by pressing and holding the SHARE and PS buttons until the controller's LED light bar flashes rapidly in white. 
+
+4. SSH into the robot and run the controller pairing script:
```

Comment:
@jyang-cpr is SSH assumed from step 1?

Reply:
Yes. I added some clarification in: https://github.com/clearpathrobotics/cpr-documentation/pull/63/commits/60600d89fd51f1dc76781755ee5d26ce36919def